packages/store-ui/src/atoms/Button/Button.tsx

```diff
@@ -2,8 +2,17 @@ import type { ButtonHTMLAttributes, ElementType } from 'react'
 import React, { forwardRef } from 'react'
 
 export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  /**
+   * Specifies the URL of the page this button click goes to.
+   */
   href?: string
+  /**
+   * Set the HTML element tag of this component.
+   */
   as?: ElementType
+  /**
+   * ID to find this component in Cypress tests.
```

Comment:
```suggestion
   * ID to find this component in test tools (e.g.: cypress).
```

Reply:
we use the testing library with jest, but stores will use cypress... there are several testing tools that can leverage this